docs/default-workflow-specs.md

```diff
@@ -0,0 +1,142 @@
+# Default Workflow Specs
+
+![alpha](assets/alpha.svg)
+
+> v2.X and after
+
+It is possible to set default workfllw specs which will be written to all workflows if the spec of interest is not set. This can be configurated through the 
```

Comment:
```suggestion
It's possible to set default workflow specs which will be written to all workflows if the spec of interest is not set. This can be configurated through the 
```